accounting/localizations/getting_started.rst

```diff
@@ -0,0 +1,491 @@
+Indeed, this one allows you to define some tags for your accounts to spread them correctly on your reports.
+For example, suppose you want to create a financial report having multiple lines but you have no way to find a rule to dispatch the accounts according their ``code`` or ``name``.
+The solution is the usage of tags, one for each report line, to spread and aggregate your accounts like you want.
+
+Like any other record, a tag can be created with the following xml structure:
+
+.. code-block:: xml
+
+    <record id="..." model="account.account.tag">
+        <!-- [Required] Specify the name to display for this tag. -->
+        <field name="name">...</field>
+
+        <!-- [Optional] Define a scope for this applicability.
+        The available keys are 'accounts' and 'taxes' but 'accounts' is the default value. -->
+        <field name="applicability">...</field>
+    </record>
+
+As you can well imagine with the usage of tags, this feature can also be used with taxes.
+
+An examples coming from the ``l10n_be`` module:
+
+.. code-block:: xml
+
+        <record id="a4000" model="account.account.template">
+            <field name="name">Clients</field>
+            <field name="code">4000</field>
+            <field name="user_type_id" ref="account.data_account_type_receivable"/>
+            <field name="chart_template_id" ref="l10nbe_chart_template"/>
+        </record>
+
+.. warning::
+
+    Don't create too much accounts: 200-300 is enough.
+
+Adding a new tax to my Chart of Accounts
+########################################
+
+To create a new tax record, you just need to follow the same process as the creation of accounts.
+The only difference being that you must use the ``account.tax.template`` model.
+
+.. code-block:: xml
+
+    <record id="..." model="account.tax.template">
+        <!-- [Required] Specify the name to display for this tax. -->
+        <field name="name">...</field>
+
+        <!-- [Required] Specify the amount.
+        E.g. 7 with fixed amount_type means v + 7 if v is the amount on which the tax is applied.
+         If amount_type is 'percent', the tax amount is v * 0.07. -->
+        <field name="amount" eval="..."/>
+
+        <!-- [Required] Set the CoA owning this tax. -->
+        <field name="chart_template_id" ref="..."/>
+
+        <!-- [Required/Optional] Define an account if the tax is not a group of taxes. -->
+        <field name="account_id" ref="..."/>
+
+        <!-- [Required/Optional] Define an refund account if the tax is not a group of taxes. -->
+        <field name="refund_account_id" ref="..."/>
+
+        <!-- [Optional] Define the tax's type.
+        'sale', 'purchase' or 'none' are the allowed values. 'sale' is the default value.
+        Note: 'none' means a tax can't be used by itself, however it can still be used in a group. -->
+        <field name="type_tax_use">...</field>
+
+        <!-- [Optional] . -->
+        <field name="tax_adjustment" eval="..."/>
```

Comment:
that's embarrassing... shouldn't we merge that boolean with the type_tax_use ??? I don't see any reason to keep it